### ISSUE TITLE
Set + implement start and stop noise

### ISSUE THREAD

**Issue by coden-s** [WhitmanCS370/alt-f4#20]
for accessibility, we should have a set sound (that probably can be changed by a user (future thing)) that plays before and after their desired sound. 
We'll need to either change how play sound works or add a new function that puts the sounds around the preexisting play sound functionality. The default may be a blank sound, but adding sound should be something that's available in settings.

Some functionality depends on having a settings flag/function.

**Comment by coden-s** [comment]
Due during week after break, before epoch due

**Comment by samuelboerner** [comment]
closed as part of restructure
